.travis.yml

```diff
@@ -15,10 +15,10 @@ before_install:
     - sudo apt-get install -qq libjpeg-dev && sudo ln -s /usr/lib/x86_64-linux-gnu/libjpeg.so /usr/lib
 
 install:
-    - pip install webapp2 requests webob db-sqlite3 nose numpy pytz pillow sphinx numpydoc coverage
+    - pip install webapp2 requests webob db-sqlite3 nose numpy==1.9.0 pytz pillow sphinx numpydoc coverage
```

Comment:
scitran/scitran is using 1.9.2; I'd be concerned if there was an incompatibility. I've pushed a commit harmonizing the version number; let's see what CI thinks.